.github/workflows/ci.yml

```diff
@@ -14,3 +14,8 @@ jobs:
       - uses: actions/checkout@v2
       - name: Build Android app
         run: bazel build android_example:lyra_android_example --config=android_arm64 --copt=-DBENCHMARK
+      - name: Upload Android APK as artifact
+        uses: actions/upload-artifact@v2
+        if: github.repository_owner == 'google'
```

Comment:
Could you please explain the reason to restrict this to the google repo?  It seems that it would be useful for others' forks as well if they are developing.

Reply:
The general reasoning was that while some find it useful, many developers also won't use the feature. This way it doesn't consume too much storage space, is always available on pull requests and on the main branch and can be enabled by hand if needed.

But if we keep the retention period short we can probably also opt to drop this restriction.

Reply:
Thanks for the explanation.  I think that's a reasonable default.